packages/SwingSet/package.json

```diff
@@ -31,6 +31,7 @@
     "@agoric/notifier": "^0.4.0",
     "@agoric/store": "^0.7.2",
     "@agoric/swing-store": "^0.7.0",
+    "@agoric/vat-data": "^0.3.1",
```

Comment:
Oh, yeah, Chip and Richard and I talked about that.

vat-data has a `devDependencies` on swingset, to support its unit tests, but not a `dependencies`. vat-data looks for names on `globalThis`, which are usually there because swingset added it to the vat environment, but vat-data doesn't actually import anything from swingset.

So I think we don't have a real package-level cycle: we have a package-level `devDependency` cycle, which I think is ok.

Reply:
I'd forgotten about the global thing. Weird way to depend de facto but not de jure ;). devDependencies are indeed ok, so all is good here. Resolving.